File: dayOne/java/LevelThree.java
public class LevelThree{

    public int [] mergeSort(int [] collectionsOne, int[] collectionsTwo){
        int [] coJoin = new int [collectionsOne.length + collectionsTwo.length];

        int index = 0;
        int indexCount = 0;
        for (int element : collectionsOne){
            coJoin[index++] = element;
            indexCount++;
        }

        for (int number : collectionsTwo){
            coJoin[indexCount++] = number;
        }
//        return coJoin;
        int count = 0;
        for (; count < coJoin.length; count++){
            int innerCount = count+1;
            for(; innerCount < coJoin.length; innerCount++){
                if(coJoin[count] > coJoin[innerCount]){
                    int swap = coJoin[count];
                    coJoin[count] = coJoin[innerCount];
                    coJoin[innerCount] = swap;
                }
            }
        }
        return coJoin;
    } 


}

File: dayOne/python/level_three.py
def merge_and_sort(collections_one, collections_two):
    merge_list = collections_one + collections_two
    for index in range(len(merge_list)):
        for counter in range(index+1, len(merge_list)):
            if(merge_list[index] > merge_list[counter]):
                swap = merge_list[index]
                merge_list[index] = merge_list[counter]
                merge_list[counter] = swap
    return merge_list

